### PR TITLE
Simplify Figure component tests/stories, move complexity to container

### LINKED ISSUE #1028: Simplify Figure component tests & stories
Currently the [Figure component fixture data](https://github.com/BBC-News/simorgh/blob/latest/src/app/components/Figure/fixtureData.jsx) file pulls in the Figure _container_. This is an issue ahead of potentially pulling the figure component out.

Furthermore, these tests make a number of assumptions based on the pre-fragment data feed, that blocks work like #801.

---

Suspect the sensible approach will be to turn the component tests into very simple snapshot tests (as many of the Figure components are already near-pure) & have more thorough container tests -- potentially using deep snapshots if that is the only way to fully check if content is rendered correctly all the way down.

### PR DESCRIPTION
**Pull request by bcmn**
Resolves #1028 

- As part of pulling out components, we realised the Figure component tests & stories were actually integration tests, utilising the Figure _container_.
- As such, this PR pulls these tests & stories showing the integration/composition of components into the container level, & creates much more granular replacements for the component level tests & stories.
- Adds stories where they were previously missing, ahead of adding components Psammead. -- i.e. Image, Caption

- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/containers/Figure/fixtureData.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import FigureContainer from '../../containers/Figure';
-import { ServiceContext } from '../ServiceContext';
+import FigureContainer from '.';
+import { ServiceContext } from '../../components/ServiceContext';
```
Comment (ChrisBAshton) [inline src/app/containers/Figure/fixtureData.jsx]: There will be conflicts with https://github.com/BBC-News/simorgh/pull/1036 🤔

**Review by ChrisBAshton — DISMISSED** [review]
Great stuff!

**Review by radiocontrolled — DISMISSED** [review]
LGTM 👍

**Comment by jamesbrumpton** [comment]
LGTM

**Review by ChrisBAshton — DISMISSED** [review]

**Review by dr3 — DISMISSED** [review]

**Review by ChrisBAshton — DISMISSED** [review]

**Review by radiocontrolled — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
Still LGTM

**Pull request merged**